### ISSUE TITLE
[TLUL agent] Support to drop VALID without READY

### ISSUE THREAD

**Issue by weicaiyang** [lowRISC/opentitan#3383]
According to TL spec, we need to have these 2 supports, which are not implemented in our agent
1. VALID can't be dropped even READY is low (I'm already working on this)
2. sender can change the contents of transition when it's not accepted, and we only consider receiver accept the item when both VALID and READY are high.

> Note that a sender may raise valid and then lower it on the following cycle, even if the message was
> not accepted on the previous cycle. For example, the sender might have some other higher priority
> task to perform on the following cycle, instead of trying to send the rejected message again.
> Furthermore, the sender may change the contents of the control and data signals when a message
> was not accepted.

> If ready is LOW, the receiver must not process the beat and the sender must not consider the
beat processed.

Related to #3354

**Comment by tjaychen** [comment]
@weicaiyang can we close this one? it looks like the enhancement is already merged.

**Comment by weicaiyang** [comment]
> @weicaiyang can we close this one? it looks like the enhancement is already merged.

the item 1 is done. I'm abort to submit another PR for item 2. Will close it once that PR is merged.

**Comment by tjaychen** [comment]
thanks!

**Comment by weicaiyang** [comment]
close via #3478 and #3585
